packages/queued-request-controller/CHANGELOG.md

```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.0]
+
 ### Added
 
 - **BREAKING**: The `QueuedRequestMiddleware` constructor now requires the `methodsWithConfirmation` param which should be a list of methods that can trigger confirmations ([#4066](https://github.com/MetaMask/core/pull/4066))
 - **BREAKING**: The `QueuedRequestController` constructor now requires the `methodsRequiringNetworkSwitch` param which should be a list of methods that need the globally selected network to switched to the dapp selected network before being processed ([#4066](https://github.com/MetaMask/core/pull/4066))
+- **BREAKING**: Clear pending confirmations (non-queued requests) after processing revokePermissions. ([#4165](https://github.com/MetaMask/controllers/pull/4165))
+- **BREAKING**: Flush the RequestQueue after processing revokePermissions. ([#4139](https://github.com/MetaMask/controllers/pull/4139))
```

Comment:
I think we should mention that QueuedRequestController requires the `SelectedNetworkController:stateChange` event and describe what happens

Reply:
yeah I mean theres a link to the PR with a lot of info about it. Im not sure how terse we need to be in changelogs. IMO it should be a very brief summary, becasue if you really care about the details of a given change, the best place to get the details is going to be in the PR.

Reply:
Ill add some more details tho.

Reply:
In general I think its been valuable to highlight precisely what is breaking in the PR and what changes the consumer needs to make to integrate those changes.